### ISSUE TITLE
[Bug?] Error: IE11 does not support 'includes' method

### ISSUE THREAD

**Issue by SotaOishi** [mobxjs/mobx-react#698]
Thanks for the update and maintaining the project.

In project（use inject and observer, Provider）,  I used this library.
When update 6.0.3 from 6.0.2, errors occured in IE11.

version
・node：10.15.3
・react：16.8.6
・mobx：4.3.1
・mobx-react：6.0.3

https://github.com/mobxjs/mobx-react/blob/master/src/utils/utils.js#L71
My guess is the issue is in the above link.

I try to replace Array.prototype.indexOf method from Array.prototype.includes method in my environment.
As a result, no errors occured.

Is this a bug or I'm I missing something?
Thanks in advance.

**Comment by FredyC** [comment]
You need to use a polyfill. Should be probably mentioned in docs, thanks for heads-up.

I recommend: https://polyfill.io

**Comment by SotaOishi** [comment]
@FredyC 
Thanks for your reply !
I'm sorry that my English isn't very good.

In actuality, i used `babel@7` .
For example
```
.babelrc

{
"presets": [
    [
      "@babel/env",
      {
        "modules": false,
        "useBuiltIns": "usage",
        "corejs": 3
      }
    ],
    "@babel/react"
  ],
   ... other
}
```

I will need to reconfirm babel config.

I understood that this library not support IE11. 
Is my understanding correct ... ?

**Comment by FredyC** [comment]
As long as you use MobX 4, it should be supported just fine. Don't use Babel for polyfill, that would be too big. That service I've mentioned detects and serves only polyfills that browser needs. You just need to add a script in your index.html.

```html
<script crossorigin="anonymous" src="https://polyfill.io/v3/polyfill.min.js?features=Array.prototype.includes"></script>
```

**Comment by sergei-startsev** [comment]
From `mobx-react` [docs](https://github.com/mobxjs/mobx-react#choosing-your-version):
>Both mobx-react 5 and 6 are compatible with **mobx 4** and 5

From `mobx` [docs](https://github.com/mobxjs/mobx#browser-support):

MobX version | Actively supported | Supported browsers
-- | -- | --
4.* | Yes (LTS) | **Any ES5 compliant browser**

I'm not sure `mobx-react 6` is compatible with `mobx 4` if it requires polyfills that `mobx` doesn't.

**Comment by mweststrate** [comment]
@sergei-startsev it is mobx-react that is compatible with mobx 4. I don't see how IE 11 compatibility logically follows from that. (mobx-react@5 is also compatible with mobx 4, and compatible with IE 11, and React 0.13, which isn't the case for 6 either). The table is just state that one can use mobx-react@6 with mobx 4. 

@FredyC I think in this specific case we can just change the code to `protoProps.indexOf(key) !== -1` and restore compatibility. (it is not a goal, but if we can remove friction by such a trivial change lets do it)

**Comment by sergei-startsev** [comment]
@mweststrate At least it should be in the list of breaking changes like a note about React support.

**Comment by mweststrate** [comment]
That is the problem with browser incompatibilities, you don't know that
you've introduced them until someone run into it 😪

Op ma 17 jun. 2019 20:58 schreef Sergei Startsev <notifications@github.com>:

**Comment by mweststrate** [comment]
Fix released as 6.0.4
